meta/change_management.md

```diff
@@ -0,0 +1,65 @@
+
+The following are the steps of phase 2:
+
+1. Gryphon conducts an "OCD pass" of all content, making it align exactly with syntax, style, and other requirements
+2. Gryphon submits a PR for review by the other 3 developers
+3. Discussion and debate take place as part phase 1 PRs, and eventually the PR is merged to `master`
+4. Gryphon builds a single *Adobe PDF file* (PDF) for the entire project
+5. Gryphon creates a PR covering the entire project
+
+## Phase 3: CQLT Review and Approval
+
+The PDF will be submitted to the CQLT for review and approval. The 6 quizzing members of the CQLT will have 1 calendar month from the date of submission to review and provide feedback on the submission. Any feedback other than general approval should be made as GitHub comments to the PR covering the entire project.
+
+The 4 developers will respond to any feedback, potentially creating subsequent versions of the output from phase 2 as necessary.
+
+At the end of the 1 month review period of phase 3, if there are no unresolved comments regarding *functional equivalence* (FE) variance, the 6 quizzing members of the CQLT will vote to approve.
+
+## Phase 4: District Ratification
+
+Following the conclusion of phase 3, Zachary will take a resulting final PDF of the whole of the project's content and submit this to all 16 active districts. Gryphon will make available a ratification PR similar to the PR from phase 3.
+
+During phase 4, anyone may provide feedback on the phase 4 PR. The 4 developers will respond to this feedback as they did for phase 3 feedback.
+
+When a *district's coordinator* (DC) confirms to his or her satisfaction that the PDF represents FE to the current rule book, that DC will notify Zachary of his or her district's ratification of the PDF. Once Zachary receives ratification notification from over 66% of the 16 districts (11 or more districts), then phase 4 is complete.
+
+At this point, the PDF will be ratified but not implemented. It will be at the CQLT's sole discretion as to when to implement the PDF as the rule book for *International Bible Quizzing* (IBQ). It will be at each district's sole discretion as to when to implement the PDF as the rule book for their district programs.
+
+## Phase 5: Ongoing Change Management
+
+This phase is the ongoing change management mechanism for this project indefinitely. This phase beings immediately after Zachary receives 11 or more district ratifications.
+
+For any changes in this phase, changes should be submitted as PRs in GitHub. These submissions can be from anyone. If someone has a change proposal but is daunted by GitHub PR management, they can contact any of the 4 original developers who can offer assistance as they are able.
```

Comment:
If changes are discussed and ratified by the CQLT "as a PR", do you want each PR to have "some limited and defined scope" ?

Reply:
I'm agnostic about this, but I'm expecting the PRs would be limited in scope naturally, so I don't want to necessarily put any restriction on them being so. There might be a time when a broad PR is useful.

Reply:
I'm fine with that. Just don't want a "pork rolling" situation, or opposite. I like your approach of not wanting to limit. If we happen to get a super wide-ranging PR, easy to request it to be split up.